source/examples/gloperate-qt-example/main.cpp

```diff
@@ -43,7 +44,7 @@ int main(int argc, char * argv[])
     UpdateManager updateManager(&environment);
 
     // Create render stage
-    auto * renderStage = new DemoStage(&environment);
+    auto * renderStage = new gloperate::DemoStage(&environment);
```

Comment:
This should get loaded by plugin?
If this is required, some of my comments above are obsolete.
Your opinion @sbusch42?